core/kazoo_apps/src/kapps_maintenance.erl

```diff
@@ -53,10 +53,11 @@
 
 -export([flush_getby_cache/0
         ,flush_account_views/0
-        ,get_all_account_views/0
+        ,get_all_account_views/0 %% tries to read from cache first
+        ,fetch_all_account_views/0 %% read views from file system (use only for debugging)
```

Comment:
What about `read_all_account_views`; I think we use `fetch` when doing a DB/AMQP type request vs `read` which usually implies a filesystem read? Thoughts?